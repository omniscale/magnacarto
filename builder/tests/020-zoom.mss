#foo[zoom=11] {
    line-width: 11;
}


#bar[zoom>=11][zoom<=14] {
    line-width: 12;
    [zoom>=15] {
        line-width: 15;
    }
}
