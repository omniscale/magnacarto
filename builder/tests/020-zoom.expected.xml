<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="foo" filter-mode="first">
    <Rule>
      <!--Zoom{=11}-->
      <MaxScaleDenominator>400000</MaxScaleDenominator>
      <MinScaleDenominator>200000</MinScaleDenominator>
      <LineSymbolizer stroke-width="11"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="bar" filter-mode="first">
    <Rule>
      <!--Zoom{11 12 13 14}-->
      <MaxScaleDenominator>400000</MaxScaleDenominator>
      <MinScaleDenominator>25000</MinScaleDenominator>
      <LineSymbolizer stroke-width="12"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="foo" srs="" status="off" maximum-scale-denominator="400000" minimum-scale-denominator="200000">
    <StyleName>foo</StyleName>
  </Layer>
  <Layer name="bar" srs="" status="off" maximum-scale-denominator="400000" minimum-scale-denominator="25000">
    <StyleName>bar</StyleName>
  </Layer>
</Map>